### ISSUE TITLE
It doesn't appear on the main screen. Does it still work for you?

### ISSUE THREAD

**Issue by SergiArias** [wp-sync-db/wp-sync-db-media-files#34]
It does nothing to main wp sync db screen. Does it happen to you as well? I understand this addon is not longer being mantained...

**Comment by gregoirepuget** [comment]
For me too...

**Comment by SergiArias** [comment]
It works, you just have to wait a little until github updates it. But you don't have another tab, it's an option in the dashboard of WP Sync. It has problems importing SVG files though...

**Comment by cjwd** [comment]
I'm having the same issue. There is no settings for media. What do you mean wait til github update its. I have github updater installed and it's not doing anything. On another note, I download the latest 1.5 release but the version in the plugin says 1.1.4b1.

**Comment by SergiArias** [comment]
It appears in the migrate tab like this:

![image](https://user-images.githubusercontent.com/13618783/30899196-2be3a94c-a35f-11e7-84ed-73f17367b331.png)

**Comment by cjwd** [comment]
That's what I'm saying, it doesn't appear there for me. I used the latest releases of both plugins.
WordPress v4.8.2
PHP 7.0.3
<img width="803" alt="screenshot 2017-09-27 17 31 29" src="https://user-images.githubusercontent.com/2500771/30938930-cdaf31b6-a3a9-11e7-8b22-5d3ededc4e48.png">

**Comment by SergiArias** [comment]
I think you have to pull from a live site in order to work...

**Comment by cjwd** [comment]
I have tried it on the live site as well. However, it would be kinda useless if it doesn't work in a local environment. Normally you want to sync between a local and a live.

**Comment by SergiArias** [comment]
Yes, I mean, you have to install it in a live site and a local site. Then, when in local site, it will appear only while pulling.

**Comment by cjwd** [comment]
What if it's a new site and you just want to export and not a pull or push. You're saying you can't do an export zip with media. I use to be able to do a zip with media. Can't remember what version that worked.

**Comment by georgeinmexico** [comment]
SergiArias, That was super helpful. 

I can confirm that I only see the "Media Files" option during a pull. 
 I can confirm that the media files addon and wp Migrate DB still work with Wordpress 4.8.3

After logging in to my live site, I select Pull and paste in the dev site URL and api key. 
Only after checking the connection to my dev site does the "Media Files" checkbox appear.

![wp-migrate-db-media-files-add-on-only-works-with-pull](https://user-images.githubusercontent.com/2537419/32627585-8bc1932c-c558-11e7-8b1e-ed0b9d714d32.png)

**Comment by artshostak** [comment]
The issue still persists where the Media Files option only comes up when using the Pull option. Any update on this?
